RedBoxMachine.java

```diff
@@ -62,20 +77,31 @@ public int searchForMovie(String title)
     */
    public boolean rent(String title)
    {
+      if (searchForMovie(title) >= 0){
```

Comment:
Instead of searching through `lib` twice in a row I would create a temporary DVD and take the value of searching through `lib` once.